initial-data-load/README.md

```diff
@@ -0,0 +1,27 @@
+# Initial data load service
+
+## Environment Variables
+
+| Env Variable                       | Required | Default Value                                                      | Description                                                                                                                                                                                                                                                                                                      |
+| ---------------------------------- | -------- | ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| NODE_ENV                           | no       | development                                                        | This environment variable is used to state in our case whether a particular environment is development, production, test or provision environment. Express uses it to alter its own default behavior. Fo example a common use-case is running additional debugging or logging code in a development environment. |
+| PORT                               | no       | 8080                                                               | The port on which the service is running.                                                                                                                                                                                                                                                                        |
+| LOG_LABEL                          | no       | oscar-initial-data-upload                                          | Title of the logs that you can see in the terminal when you run the service.                                                                                                                                                                                                                                     |
+| AZURE_STORAGE_CONNECTION_STRING    | yes      | -                                                                  | Default Azurite connection string from documentation used for a local development.                                                                                                                                                                                                                               |
```

Comment:
I would not describe it as default. It's simply 'Connection string for Azure Storage (in cloud or local Azurite)

Reply:
I fixed this.